.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: release
+run-name: Release
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+"
+
+jobs:
+  tests:
+    uses: ./.github/workflows/test.yml
+    secrets: inherit
+  pypi-wheel:
+    needs: [ tests ]
+    uses: ./.github/workflows/pypi-wheel-builds.yml
+    secrets: inherit
+  upload_pypi:
+    name: Upload to pypi repository
+    needs: [ tests, pypi-wheel ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          name: artifact
+          path: dist
+      - name: Publish package to TestPyPI
+        if: ${{ contains(github.ref, 'rc') }}
+        # TODO: Maybe we can move this to main PyPI since it is marked rc
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          repository_url: https://test.pypi.org/legacy/
```

Comment:
Do we want to move the `rc` to the main PyPI? Since they are `rc` releases, they will not be updated automatically unless specifed, e.g. check the [TestPyPI history](https://test.pypi.org/project/spglib/#history).

Reply:
I think it's enough to upload the `rc` to TestPyPI if there are no demands to upload to the main PyPI.